lib/la-arena/src/lib.rs

```diff
@@ -77,6 +77,16 @@ impl<T> fmt::Debug for Idx<T> {
     }
 }
 
+impl<T: 'static> rustc_index::vec::Idx for Idx<T> {
```

Comment:
`la-arena` is a published crate with stable API, so I'm not sure we want to add this impl and dependency

Reply:
Is it acceptable behind a feature gate?

Reply:
Problem is that every bump of `rustc_index` is a breaking change for the `la-arena` crate, so I'd prefer to remove it entirely to avoid churn. We should be able to create a newtype in the r-a crates that need it.